src/Contracts/PageFactory.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Astrotomic\Stancy\Contracts;
+
+use Astrotomic\Stancy\Models\Page;
+
+interface PageFactory
+{
+    public function make(array $data = [], ?string $page = null): Page;
```

Comment:
Returns `Contracts\Page` instead of the real class. Please alias it as `PageContract` in the use statement.

Reply:
Whoops, I missed this one 😅 Thanks